### PR TITLE
Baseline UI

### PR DESCRIPTION
**Pull request by zanemcca**
## Completed
- Add a panel with three sub panels to show the tasks for today, tomorrow and the next day
- Add a panel that shows the tasks in a pre-specified Asana project

## Not Completed
- Mobile friendly responsiveness

### CODE REVIEW

**Comment by zanemcca** [comment]
Preview

**Comment by priyadarshy** [comment]
Adding your e-mail for context: 
```
Hey Ashutosh,

I have a PR ready with the baseline UI recreation that will be the foundation for the rest of the week. I would love to get your take on it.

I have added you as a collaborator to my Github repo. If you did not get the link here is a copy https://github.com/zanemcca/task_planner_4000/invitations.
Here is the link for the first PR https://github.com/zanemcca/task_planner_4000/pull/1. If you don't care much about the code I have included a preview image of the UI and some details about what is completed so far in the PR description.

Cheers,
Zane
```

Scaffolding has the correct structure! 

I would not spend anytime working on mobile responsiveness for this project.

**Comment by priyadarshy** [comment]
Can you also add my colleagues @BrunoBernardino and @bone-saw as collaborators so we can get a better sense of collaborating with you!

**Comment by zanemcca** [comment]
@priyadarshy I've added both of them. Please let me know if they have issues with the invites.

**Comment by zanemcca** [comment]
[Recreate Baseline UI](https://trello.com/c/S2QLnwXV/4-recreate-baseline-ui)

**Comment by priyadarshy** [comment]
Since this looks good, I'll go ahead and merge it in (since thats how we typically do it). We also often leave a super short and informal Loom video with PRs that have something that can be show off. 

Here's some examples:
- https://www.loom.com/share/845646d7af7148b8afb41fd6412e3c46
- https://www.loom.com/share/2753417295594a1d86ea1b5671ce0566

**Comment by zanemcca** [comment]
@priyadarshy thanks for the the review!

@BrunoBernardino once you see this please share your preferred email and I can add you to the Trello board I got started. 

https://trello.com/b/8UX1csVs/sunsama-project

@bone-saw and @priyadarshy I've already added you to the board let me know if you have trouble getting connected.

**Comment by zanemcca** [comment]
@priyadarshy I appreciate the loom examples. I will do one up for the next PR

File: package.json [hunk 1]
```diff
@@ -10,6 +10,9 @@
     "@types/node": "^12.0.0",
     "@types/react": "^16.9.0",
     "@types/react-dom": "^16.9.0",
+    "antd": "^3.26.8",
```
Comment (BrunoBernardino) [inline package.json]: It's cool to see you added something to speed you up with the UI.

File: src/App.tsx [hunk 2]
```diff
@@ -1,24 +1,75 @@
...
+  },
+  personal: {
+    title: 'personal',
+    color: 'blue'
+  }
+}
+
 const App = () => {
+  const today = moment()
+  const schedules: IDailySchedule[] = [{
```
Comment (BrunoBernardino) [inline src/App.tsx]: **It honestly does not matter in this context**, but if this was a real/production feature I'd say:

Setting up this array here will make it so it's computed on every render, vs just once or initially (by moving that code outside, or into a `componentDidMount` or a `useEffect` hook), which can slow down performance with more tasks/data, so I'd suggest making that change.

File: src/components/__tests__/Asana.test.tsx [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import renderer from 'react-test-renderer';
+
+import Asana, { IAsanaProps } from '../Asana';
+import moment from 'moment';
+import React from 'react';
+
+it('should find and render tasks when project is given', () => {
```
Comment (BrunoBernardino) [inline src/components/__tests__/Asana.test.tsx]: These aren't really necessary for this project, but I appreciate the snapshot tests!

**Comment by bone-saw** [comment]
Reviewed screenshot and looked over code carefully. Didn't pull down.

I'm late to the party, but just wanted to say looks good to me for a foundation.

File: src/App.tsx (hunk 2 above)
Comment (zanemcca) [reply src/App.tsx]: That is a very fair point. My plan is to move everything into a state once I had forms setup to change it. However I probably should have cleaned it up before pushing the PR. Thanks for catching it.

File: src/components/__tests__/Asana.test.tsx (hunk 3 above)
Comment (zanemcca) [reply src/components/__tests__/Asana.test.tsx]: Noted thanks!

File: src/App.tsx (hunk 2 above)
Comment (BrunoBernardino) [reply src/App.tsx]: That makes sense, thanks for clarifying! Like I said, it doesn’t matter much here, for the sake of this demo.

**Pull request merged**